exemplars/main/rna.xml

```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
-<!-- Copyright © 1991-2024 Unicode, Inc.
-For terms of use, see http://www.unicode.org/copyright.html
-SPDX-License-Identifier: Unicode-3.0
-CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
--->
-<ldml>
-	<identity>
-		<version number="$Revision$"/>
-		<language type="rna"/>
-	</identity>
-	<layout>
-		<orientation>
-			<characterOrder>left-to-right</characterOrder>
-			<lineOrder>top-to-bottom</lineOrder>
-		</orientation>
-	</layout>
-	<characters>
-		<exemplarCharacters draft="unconfirmed">[a b c d e f g h i j k m n o p r s t u v w y z]</exemplarCharacters>
```

Comment:
Why was this deleted?

Reply:
OK, I can't find what this was supposed to be, so fine to remove.

Reply:
The file itself has a deprecated language code, 'rna', without any replacement alias provided by ISO.